server/statistics/store.go

```diff
@@ -265,3 +265,43 @@ func (r *RollingStoreStats) GetInstantLoad(k StoreStatKind) float64 {
 	}
 	return 0
 }
+
+// SlowStoresStats is a cached statistics for the slow store.
+type SlowStoresStats struct {
```

Comment:
I think the struct can  move to slow_stat.go.

Reply:
Okay.